config/locales/en/generic.yml

```diff
@@ -124,6 +124,9 @@ en:
   notice: "Notice"
   notices: "Notices"
   dismiss: "Dismiss"
+  dismiss_all_notices: "Dismiss All"
+  dismiss_all_notices_confirmation: "Are you sure want to dismiss all notices?
+    <br/>This action cannot be undone."
```

Comment:
In html5, the correct markup is `<br>`, not `<br/>`. Strange, but true. In general, it is much better to avoid html markup in translations. It just confuses translators and introduces potential security problems. Sadly, it is still used in various places.

In this specific case, I think maybe we should create a separate key for "This action cannot be undone."